### PR TITLE
Fix Kafka config

### PR DESCRIPTION
**Pull request by xakep139**

### CODE REVIEW

File: src/VStore/Kafka/ConsumerWrapper.cs [hunk 1]
```diff
@@ -38,22 +40,23 @@ public virtual void Dispose()
...
-        private static Dictionary<string, object> CreateConsumerConfig(string brokerEndpoints, string groupId)
+        private static Dictionary<string, object> CreateConsumerConfig(KafkaOptions kafkaOptions, string groupId)
             => new Dictionary<string, object>
                 {
-                    { "bootstrap.servers", brokerEndpoints },
+                    // https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
+                    { "bootstrap.servers", kafkaOptions.BrokerEndpoints },
                     { "api.version.request", true },
                     { "group.id", !string.IsNullOrEmpty(groupId) ? groupId : Guid.NewGuid().ToString() },
-                    { "socket.blocking.max.ms", 1 },
```
Comment (xakep139) [inline src/VStore/Kafka/ConsumerWrapper.cs]: This causes AMS-817

File: src/VStore/Kafka/EventSender.cs [hunk 2]
```diff
@@ -26,17 +26,18 @@ public EventSender(KafkaOptions kafkaOptions, ILogger<EventSender> logger)
 
             var producerConfig = new Dictionary<string, object>
                 {
+                    // https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md
                     { "bootstrap.servers", kafkaOptions.BrokerEndpoints },
                     { "api.version.request", true },
-                    { "socket.blocking.max.ms", 1 },
```
Comment (xakep139) [inline src/VStore/Kafka/EventSender.cs]: This also causes AMS-817

File: src/VStore/Kafka/ConsumerWrapper.cs [hunk 3]
```diff
@@ -38,22 +40,23 @@ public virtual void Dispose()
...
-                    { "enable.auto.commit", false },
-                    { "fetch.wait.max.ms", 5 },
-                    { "fetch.error.backoff.ms", 5 },
-                    { "fetch.message.max.bytes", 10240 },
-                    { "queued.min.messages", 1000 },
+                    { "enable.auto.commit", kafkaOptions.Consumer.AutoCommit },
```
Comment (denisivan0v) [inline src/VStore/Kafka/ConsumerWrapper.cs]: `EnableAutoCommit`?

Comment (xakep139) [reply src/VStore/Kafka/ConsumerWrapper.cs]: Fixed in https://github.com/2gis/nuclear-vstore/pull/121/commits/1e864f5843b988f023724534f2786056d26a97ac

**Pull request merged**